content/en/_redirects

```diff
@@ -13,55 +13,55 @@
 /blog/announcing_kubeflow_0.2/         https://medium.com/kubeflow/kubeflow-0-2-offers-new-components-and-simplified-setup-735e4c56988d
 
 /docs/guides/components/components     /docs/components/
-/docs/guides/pipelines/deploy-pipelines-service/  /docs/pipelines/pipelines-quickstart/
+/docs/guides/pipelines/deploy-pipelines-service/  /docs/components/pipelines/pipelines-quickstart/
 
 # Removed the "guides" section from the doc URLs.
 /docs/guides/*  /docs/:splat
 
 # Merged duplicate page pipelines-samples.md into build-pipeline.md
-/docs/pipelines/pipelines-samples/	   /docs/pipelines/build-pipeline/
+/docs/components/pipelines/pipelines-samples/	   /docs/components/pipelines/build-pipeline/
```

Comment:
I think we don't need to add `/components` in src.
They should always redirect from the original path to the final path we are updating them to.

Reply:
also we need a `/docs/pipelines /docs/pipelines/components` redirection for links pointing to KFP documentation in  other places in the internet

Reply:
@Bobgy please check this [commit](https://github.com/kubeflow/website/pull/2505/commits/1e9968118eee06bf6f56de72958267296112d02c) and LMKWYT